### ISSUE TITLE
Support macOS target

### ISSUE THREAD

**Issue by hotip** [mori-atsushi/insetsx#57]
adding this lib to https://github.com/JetBrains/compose-multiplatform/tree/master/examples/chat

got this error:

```
Could not resolve all dependencies for configuration ':shared:macosArm64CompileKlibraries'
```

maybe you can add a target for it?  just leave empty implementation

**Comment by hotip** [comment]
```
    macosX64 {
    }
    macosArm64 {
    }
```

**Comment by mori-atsushi** [comment]
I have released v0.1.0-alpha08, which supports  macOS🎉
https://github.com/mori-atsushi/insetsx/releases/tag/v0.1.0-alpha09
